src/test/issues.c2.test.ts

```diff
@@ -91,6 +92,21 @@ describe("Issue Tests", () => {
         );
     });
 
+    it("#2574", () => {
+        const project = convert();
+        const usesDefaultExport = query(project, "usesDefaultExport");
+        const sig = usesDefaultExport.signatures?.[0];
+        ok(sig, "Missing signature for usesDefaultExport");
+        const param = sig.parameters?.[0];
+        ok(param, "Missing parameter");
+        equal(param.name, "param", "Incorrect parameter name");
+        const paramType = param.type as ReferenceType | undefined;
+        ok(paramType, "Parameter type is not a reference type or undefined");
+        equal(paramType.type, "reference", "Parameter is not a reference type");
```

Comment:
`as` doesn't check anything, so this is misleading, you can just check that the type's `type` member is `reference` to know it is a reference type.

```suggestion
        const paramType = param.type;
        equal(paramType?.type, "reference", "Parameter type is not a reference type");
```